userspace/otpilot/src/gpio_processor.rs

```diff
@@ -129,6 +129,11 @@ impl GpioProcessor {
             }
         }
 
+        let sys_rstmon_n = gpio::get().consume_event(GpioPin::SYS_RSTMON_N);
```

Comment:
Hmm.  Are we certain that it's OK to ignore a reset event here?  Is it possible that this logic will cause us to ignore an intended reset?

(This is why the guy I learned most of what I know about distributed systems from _hates_ edges (i.e. events) -- he prefers levels.)

Reply:
We don't have anything to do for SYS_RSTMON_N yet. You could argue that we might not want ANY handling for this GPIO at this time. However, I found it important to provide all the plumbing so that handling could be easily added later once we actually do attestation.

Reply:
:+1: